src/worker/views/editAwards.tsx

```diff
@@ -0,0 +1,104 @@
+import { PLAYER } from "../../common";
+import type { UpdateEvents, ViewInput } from "../../common/types";
+import { idb } from "../db";
+import { g } from "../util";
+
+const updateAwards = async (
+	inputs: ViewInput<"editAwards">,
+	updateEvents: UpdateEvents,
+) => {
+	if (!g.get("godMode")) {
+		// https://stackoverflow.com/a/59923262/786644
+		const returnValue = {
+			errorMessage: "You can't edit awards unless you enable God Mode.",
+		};
+		return returnValue;
+	}
+	const season = inputs.season;
+	let awards = await idb.getCopy.awards({
+		season,
+	});
+
```

Comment:
Would be nice to default to last season if we know there are no awards for the current season, like:

```
if (!awards) {
    if (g.get("season") === season && g.get("phase") <= PHASE.PLAYOFFS) {
        season -= 1;
        awards = await idb.getCopy.awards({
		season,
	});
    }
}
```

(This is done after checking for the presence of `awards` in case some weird situation happens where there actually is an awards object before this season's playoffs end)